### PR TITLE
Template improvements

### PR DESCRIPTION
**Pull request by emanuelschuetze**
* Fix translations in group detail template
* Agenda list slide improvements

### CODE REVIEW

File: openslides/users/static/templates/users/group-detail.html [hunk 1]
```diff
@@ -6,14 +6,15 @@
...
   </div>
 </div>
 
 <div class="details">
   <p class="lead" translate>Permissions:</p>
-  <ul ng-repeat="perm in group.permissions">
-    <li>{{ perm }}
+  <ul ng-repeat="permission in groupPermissionNames">
+    <li>{{ permission | translate }}
+
```
Comment (normanjaeckel) [inline openslides/users/static/templates/users/group-detail.html]: Proposal: Remove empty line here.

**Comment by normanjaeckel** [comment]
One proposal. Then it can be merged.

File: openslides/users/static/templates/users/group-detail.html (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/users/static/templates/users/group-detail.html]: done

**Comment by emanuelschuetze** [comment]
All done.

**Pull request merged**